sktime/clustering/base/base.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+"""Base classes for clustering."""
+
+__author__ = ["Christopher Holder", "Tony Bagnall"]
+__all__ = [
+    "BaseCluster",
+    "ClusterMixin",
+    "BaseClusterCenterInitializer",
+    "BaseClusterAverage",
+    "BaseApproximate",
+    "Init_Algo",
+    "Averaging_Algo",
+    "Averaging_Algo_Dict",
+    "Init_Algo_Dict",
+]
+
+import pandas as pd
+from sktime.base import BaseEstimator
+from sktime.utils.data_processing import from_nested_to_2d_array
+from sktime.clustering.base.base_types import Numpy_Array, Numpy_Or_DF
+from typing import Union, Mapping
+
+
+class BaseCluster(BaseEstimator):
+    def __init__(self):
+        self._is_fitted = False
+        super(BaseCluster, self).__init__()
+
+    def fit(self, X: Numpy_Or_DF) -> None:
+        """
+        Method that is used to fit the clustering algorithm
+        on the dataset X
+
+        Parameters
+        ----------
+        X: Numpy array or Dataframe
+            sktime data_frame or numpy array to train the model on
+
+        Returns
+        -------
+        self
+            Fitted estimator
+        """
+        self._is_fitted = False
```

Comment:
I don't think we need this here. As far as I know, it was only necessary for forecasters as a temporary fix, it will be set to False during construction in the `BaseEstimator`.
```suggestion
```